glusterapi/peer.py

```diff
@@ -27,19 +27,19 @@ def peer_add(self, host, metadata=None, zone=""):
 
     def peer_remove(self, peerid):
         """
-        Gluster Peer Remove
+        Gluster Peer Remove.
 
         :param host: (string) Hostname or IP
```

Comment:
can you change this function comment, we cannot remove peer from Hostname or IP, a peer can be removed only by using peer id.

Reply:
done.